README.md

```diff
@@ -51,7 +51,17 @@ Contains the JSON schemas used to validate the sources in the integration tests.
 
 ## GTFS Realtime Data Structure
 
-Still in progress. [Please review the working document to provide your feedback](https://docs.google.com/document/d/1Mlz3AXHItInitsOEAKKi8hZV9d3t3gpFbyIZGm67ESU/edit#heading=h.88f252jw0urf).
+|     Field Name     |  Required from users  |                                                                              Definition                                                                             |
+|:------------------:|:---------------------:|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------:
+| MDB Source ID      | No - system generated | Unique numerical identifier.      |   |   |
+| Data Type          | Yes                   | The data format that the source uses: GTFS.                                                                                                            |   |   |
```

Comment:
It should be `GTFS_RT` here. Also, to be extra clear, it should be `tools.constants.GTFS_RT` or `"gtfs_rt"`. This is also true for GTFS Schedules sources with `tools.constants.GTFS` or `"gtfs"`
```suggestion
| Data Type          | Yes                   | The data format that the source uses: GTFS_RT.                                                                                                            |   |   |
```